Epam.TAF.BDD/Steps/Pages/HeaderLinksPagesSteps.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Epam.TAF.Core.Enums;
+using Epam.TAF.Web.PageObgects.Pages;
+using NUnit.Framework;
+using TechTalk.SpecFlow;
+
+namespace Epam.TAF.BDD.Steps.Pages
+{
+    [Binding]
+    public class HeaderLinksPagesSteps
+    {
+        AboutPage AboutPage => new();
+        CareersPage CareersPage => new();
+        InsightsPage InsightsPage => new();
+        ServicesPage ServicesPage => new();
+
+        [Then(@"I check that '(Services|Insights|About|Careers)' link page is opened")]
+        public void ThenICheckThatLinkPageIsOpened(HeaderLinkEnums headerLink)
```

Comment:
This step is okay for now, but for future it would be better to split them by separate steps files to be able to reuse.
So your current method will look like this:
 switch (headerLink)
            {

                case HeaderLinkEnums.Services:
                    new ServicesPageSteps().ThenICheckThatServicesPageIsOpened();
                    break;
                case HeaderLinkEnums.Insights:
                    new InsightsPageSteps().ThenICheckThatInsightsPageIsOpened();
                    break;
                    }